src/components/inputs/SearchSelect.jsx

```diff
@@ -432,22 +433,24 @@ const SearchSelect = React.createClass({
             )
           }
         </div>
-        <div
-          className={classNames('SearchSelect-optionsList', `SearchSelect-optionsList--${isListOpen ? 'open' : 'closed'}`)}
-        >
-          {_.map(filteredOptions, (option, i) => {
-            let render = option.isGroup ? this._renderGroup : this._renderOption;
-            return render(option, i);
-          })}
-        </div>
+        {isListOpen &&
+          <div
+            className={classNames('SearchSelect-optionsList', `SearchSelect-optionsList--${isListOpen ? 'open' : 'closed'}`)}
```

Comment:
You can remove the state-dependent className since it will now always be `open`. Also remove the CSS if there is any.